### PR TITLE
Feature/2609 new log format for stdout

### PR DESCRIPTION
**Pull request by kzangeli**
Fixes issue #2609

### CODE REVIEW

File: scripts/orion [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```
Comment (fgalan) [inline scripts/orion]: Name 'orion' for this file doesn't provide hints about in which way this script is different from just calling contextBroker...

Maybe 'contextBrokerFg' would be a better name?

Comment (kzangeli) [inline scripts/orion]: actually, the suffix is missing as well ...
As we now are 'destroying' the broker, it cannot really be called by hand anymore, the script needs a short name, easy to use. That it invokes 3 cli by default we will just have to remember.
I would propose orion.sh.

[ But, we can always give it an unusable name and then all create aliases for it instead. I just don't see the reason for it. This script is supposed to replace the alias ]

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1,2 +1,5 @@
 - Fix: threadpool mode not actually used for custom notifications, even if -notificationMode is set so (#2622)
 - Fix: -logAppend as default in configuration for RPM package (#2637)
+- Fix: Default log output for stdout is now the same format as in the log file. Use new CLI option '-logForHumans' to nodify this. (Issue #2609)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: Line 2 should be removed. Line 3 is probably redundant as -logForHumans is already mentioned in line 1.

File: doc/manuals/admin/cli.md [hunk 3]
```diff
@@ -152,3 +152,5 @@ The list of available options is the following:
 -   **-disableCustomNotifications**. Disabled NGSIv2 custom notifications. In particular:
     * `httpCustom` is interpreted as `http`, i.e. all sub-fields except `url` are ignored
     * No `${...}` macro substitution is performed.
+-   **-logToScreen**. To make the broker send traces to standard out (stdout) as well as to the log file.
```
Comment (fgalan) [inline doc/manuals/admin/cli.md]: Line to remove

File: src/app/contextBroker/contextBroker.cpp [hunk 4]
```diff
@@ -389,12 +391,15 @@ PaArgument paArgs[] =
...
   { "-statNotifQueue", &statNotifQueue, "STAT_NOTIF_QUEUE", PaBool, PaOpt, false, false, true, STAT_NOTIF_QUEUE  },
 
-  { "-logSummary",     &lsPeriod,       "LOG_SUMMARY_PERIOD", PaInt,PaOpt,   0,     0,     ONE_MONTH_PERIOD, LOG_SUMMARY_DESC },
+  { "-logSummary",     &lsPeriod,       "LOG_SUMMARY_PERIOD", PaInt,  PaOpt, 0,     0,     ONE_MONTH_PERIOD, LOG_SUMMARY_DESC },
   { "-relogAlarms",    &relogAlarms,    "RELOG_ALARMS",       PaBool, PaOpt, false, false, true,             RELOGALARMS_DESC },
 
   { "-strictNgsiv1Ids",             &strictIdv1,      "CHECK_ID_V1",           PaBool, PaOpt, false, false, true, CHECK_v1_ID_DESC      },
   { "-disableCustomNotifications",  &disableCusNotif, "DISABLE_CUSTOM_NOTIF",  PaBool, PaOpt, false, false, true, DISABLE_CUSTOM_NOTIF  },
 
+  { "-logToScreen",   &logToScreen,     "LOG_TO_SCREEN",      PaBool, PaOpt, false, false, true,             LOG_TO_SCREEN_DESC  },
```
Comment (fgalan) [inline src/app/contextBroker/contextBroker.cpp]: I understand this should be removed.

File: src/app/contextBroker/contextBroker.cpp [hunk 5]
```diff
@@ -1615,12 +1620,16 @@ int main(int argC, char* argV[])
 
 
   //
-  // If option '-fg' is set, print traces to stdout as well, otherwise, only to file
+  // If option '-logToScreen' is set, print traces to stdout as well, otherwise, only to file
```
Comment (fgalan) [inline src/app/contextBroker/contextBroker.cpp]: It seems that the change in this line should be rolled back

File: src/app/contextBroker/contextBroker.cpp [hunk 6]
```diff
@@ -1615,12 +1620,16 @@ int main(int argC, char* argV[])
...
   //
-  if (paIsSet(argC, argV, "-fg"))
+  if (paIsSet(argC, argV, "-logToScreen") || paIsSet(argC, argV, "-fg"))
   {
     paConfig("log to screen",                 (void*) true);
-    paConfig("screen line format",            (void*) "TYPE@TIME  FILE[LINE]: TEXT");
+
+    if (paIsSet(argC, argV, "-logForHumans"))
```
Comment (fgalan) [inline src/app/contextBroker/contextBroker.cpp]: Style: `{` etc.

File: test/functionalTest/cases/0000_cli/bool_option_with_value.test [hunk 7]
```diff
@@ -84,5 +84,7 @@ Usage: contextBroker  [option '-U' (extended usage)]
                       [option '-relogAlarms' (log messages for existing alarms beyond the raising alarm log message itself)]
                       [option '-strictNgsiv1Ids' (additional checks for id fields in the NGSIv1 API)]
                       [option '-disableCustomNotifications' (disable NGSIv2 custom notifications)]
+                      [option '-logToScreen' (log to screen)]
```
Comment (fgalan) [inline test/functionalTest/cases/0000_cli/bool_option_with_value.test]: To be removed.

File: src/app/contextBroker/contextBroker.cpp (hunk 4 above)
Comment (kzangeli) [inline src/app/contextBroker/contextBroker.cpp]: Fixed in df66109

File: CHANGES_NEXT_RELEASE (hunk 2 above)
Comment (kzangeli) [inline CHANGES_NEXT_RELEASE]: Fixed in df66109. Two lines kept though

File: src/app/contextBroker/contextBroker.cpp (hunk 6 above)
Comment (kzangeli) [inline src/app/contextBroker/contextBroker.cpp]: Fixed in 1bff493

**Comment by fgalan** [comment]
LGTM

**Pull request merged**